main.go

```diff
@@ -142,15 +142,21 @@ func main2() error {
 			// Get request body
 			if op.RequestBody != nil && op.RequestBody.Value.Content["application/json"] != nil {
 				if schema := op.RequestBody.Value.Content["application/json"].Schema; schema != nil {
-					def := schemaRefParse(schema, "Body")
-					if def.Name != "Body" {
+					def := schemaRefParse(schema, "")
+					if def.Name != "" {
```

Comment:
Perhaps a comment here saying what the lack of name implies might be good?

Reply:
It's actually doing the same thing as before, but `"Body"` is unnecessary and actually would break if the user defines a component called `Body`. Therefore an empty string felt much safer.